.gitlab-ci.yml

```diff
@@ -42,3 +43,57 @@ test:
       - "out/cover.html"      
       - "out/tagent"
     expire_in: 1 year
+
```

Comment:
Is this going to be an issue when we try to merge back into v3.3/develop?  I think this indicates that this fork's master is out of sync.

Reply:
It was informed that scan job has been added in the pipeline. In v3.2 it was not there so we have referred v3.3/develop and included as we thought it may needed. This change wont be included, we will revert this back.